uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt

```diff
@@ -404,17 +393,13 @@ class BookingRequestView @JvmOverloads constructor(context: Context,
     }
 
     override fun handlePaymentDetailsUpdate() {
-        presenter.setBookingEnablement(bookingRequestPassengerDetailsWidget.allFieldsValid())
+        // not used
```

Comment:
Shall we remove the handlePaymentDetailsUpdate if it is no longer in use?

Reply:
cannot do it, this interface is used in other places and it has multiple methods. We only do not use that one